### DISCUSSION TITLE
Customize which views are available

### DISCUSSION THREAD

**Discussion by davish** [davish/obsidian-full-calendar#353] [Feature Requests]
Right now, there's hardcoded views for desktop (month, week, day, week list) and mobile (3 day, 1 day, week list). This should be configurable by the user in settings.

**Comment by DavidBurela** [comment]
The important one that I would like to see, is a month view on Mobile

**Comment by YouFoundJK** [comment]
This is now possible in my fork via [workspaces](https://youfoundjk.github.io/plugin-full-calendar/views/workspaces/).
